changelogs/unreleased/3125-pranavgaikwad

```diff
@@ -0,0 +1 @@
+Restore progress reporting
```

Comment:
I'd suggest something like "Progress reporting added for restores" for a changelog entry.